apps/api/src/registrant/registrant.controller.ts

```diff
@@ -51,12 +57,31 @@ export class RegistrantController {
     return registrants;
   }
 
+  @Roles(Role.Admin, Role.User)
+  @UseGuards(AuthGuard, RoleGuard)
+  @UseInterceptors(ClassSerializerInterceptor)
+  @Get('/mass-email')
```

Comment:
Maybe change this to /mass-email-history for clarity in debugging?

Reply:
updated.